photon-core/src/main/java/org/photonvision/common/dataflow/networktables/NTDataPublisher.java

```diff
@@ -146,15 +146,22 @@ public void accept(CVPipelineResult result) {
                             List.of(),
                             result.inputAndOutputFrame);
         else acceptedResult = result;
-        var now = WPIUtilJNI.now();
-        var captureMicros = MathUtils.nanosToMicros(acceptedResult.getImageCaptureTimestampNanos());
+        var now = NetworkTablesJNI.now();
+        var captureMicros = MathUtils.nanosToMicros(result.getImageCaptureTimestampNanos());
+
+        var offset = NetworkTablesManager.getInstance().getOffset();
+
+        // Transform the metadata timestamps from the local nt::Now timebase to the Time Sync Server's
+        // timebase
         var simplified =
                 new PhotonPipelineResult(
-                        acceptedResult.sequenceID,
-                        captureMicros,
-                        now,
-                        TrackedTarget.simpleFromTrackedTargets(acceptedResult.targets),
-                        acceptedResult.multiTagResult);
+                        result.sequenceID,
```

Comment:
result vs acceptedresult?

Reply:
ur diff is outdated

Reply:
which is why it was marked as resolved :)